### DISCUSSION TITLE
Unable to get page link in query for bigcommerce category/product link

### DISCUSSION THREAD

**Discussion by asimonok** [wp-graphql/wp-graphql#2302] [General] [answered]
Hi there,
We have Advanced Custom Field and BigCommerce for Wordress plugins and have to map bigcommerce product/category links in menus and custom fields. But in queries these fields are `null` if it was set link for bigcommerce product or category.
I've attached screenshots how mapping works in menu now for more details what I mean.
The same for fields which were set via advance custom field and were mapped on bigcommerce category or product.
Is it possible to handle that and start getting entities like bigcommerce category or product?
<img width="1110" alt="Снимок экрана 2022-03-18 в 18 41 08" src="https://user-images.githubusercontent.com/15867703/159035465-863a34af-0ab7-4d72-b1d9-864c72678ea2.png">
<img width="989" alt="Снимок экрана 2022-03-18 в 18 42 56" src="https://user-images.githubusercontent.com/15867703/159035443-691d4b3b-c90b-4469-a9b5-d9eca064c0ce.png">

Wordpress version: 5.9.2
Active plugins:
<img width="647" alt="Снимок экрана 2022-03-18 в 18 47 47" src="https://user-images.githubusercontent.com/15867703/159036222-4604e6eb-aa20-4375-ad4d-2f1c8b5e49c3.png">

**Comment by justlevine** [comment, marked as answer]
Are you [exposing those cpt and taxonomy types to the schema](https://www.wpgraphql.com/docs/custom-post-types/#filtering-an-existing-post-type)?

(ps, it also wont be `Category`, but `ProductCategory` or whatever you name it, since the former is the taxonomy used for `Posts`.
